include/env.bash

```diff
@@ -167,6 +169,8 @@ PERISCOPE_DB_ENV_DB - Name of the Autoscale database
 PERISCOPE_DB_ENV_PASS - Password for the Autoscale database authentication
 PERISCOPE_DB_ENV_SCHEMA - Used schema in the Autoscale database
 PERISCOPE_DB_ENV_USER - User for the Autoscale database authentication
+PERISCOPE_DB_ENV_SSL - enable of ssl certification or not
```

Comment:
I would describe this variable as: `Enable SSL database connection for Periscope`